### DISCUSSION TITLE
Numbers are now strings in station data

### DISCUSSION THREAD

**Discussion by jfbauer432** [weather-gov/api#335] [Q&A] [answered]
A bit earlier today, I discovered that the numeric data in the latest station data (e.g. https://api.weather.gov/stations/KBWI/observations/latest) are now strings.

i.e. prior
`        "windSpeed": {
            "value": 1.5,
            "unitCode": "unit:m_s-1",
            "qualityControl": "qc:V"
        },
`
now

`        "windSpeed": {
            "value": "9.359999999999999999999999999993",
            "unitCode": "unit:km_h-1",
            "qualityControl": "qc:V"
`

**Comment by sramsay** [comment]
As far as I can tell, this affects only the station data (but it's pretty catastrophic for anyone talking to the API through a strongly typed language).

**Comment by StephenClouse** [comment]
This is an unintentional change and we're looking into it.

**Comment by StephenClouse** [comment]
We have a hotfix for this issue scheduled for deployment on Monday 6/22.

**Comment by StephenClouse** [comment, marked as answer]
We have pushed this hotfix and it looks like things are back to how they should be.

**Comment by robertbcalhoun** [comment]
Thank you! I've stopped seeing JSON parse errors as the hotfix rolled out.
![image](https://user-images.githubusercontent.com/5921027/85597684-d8bda900-b618-11ea-9aa9-44d2b6bb82ab.png)
(Times in EDT, because Kibana likes to use my local TZ.)
